### ISSUE TITLE
Version 1.1.1 incompatible with .NET Framework

### ISSUE THREAD

**Issue by doug-shontz** [cloudinary/CloudinaryDotNet#86]
We have a shared C# library that encapsulates a good bit of business logic from a number of vendors...Cloudinary being one of them.  We have been on the 1.0.x line with no issues.  We are testing the 1.1.x line and are able to update to 1.1.0.  But, in attempting to update to 1.1.1, NuGet is reporting that the package is incompatible with .NET Framework 4.7.1.

Is this intentional?  We're not at a place where we can move strictly to .NET Standard.  I could probably just create a .NET Standard library just to house Cloudinary if necessary, but it seems that a breaking change like that would have been noted in the changelog?

Any assistance would be greatly appreciated.

**Comment by roeeba** [comment]
Hi @doug-shontz, I apologize for the delayed reply. We're checking it now and will update ASAP.

**Comment by roeeba** [comment]
Hi @doug-shontz. Thank you for your patience. We're unable to reproduce the issue on our side. Here's the output we get:

```
Attempting to gather dependency information for package 'CloudinaryDotNet.1.1.1' with respect to project 'ConsoleApp1', targeting '.NETFramework,Version=v4.7.1'
Gathering dependency information took 714.88 ms
Attempting to resolve dependencies for package 'CloudinaryDotNet.1.1.1' with DependencyBehavior 'Lowest'
Resolving dependency information took 0 ms
Resolving actions to install package 'CloudinaryDotNet.1.1.1'
Resolved actions to install package 'CloudinaryDotNet.1.1.1'
Retrieving package 'CloudinaryDotNet 1.1.1' from 'nuget.org'.
GET https://api.nuget.org/v3-flatcontainer/cloudinarydotnet/1.1.1/cloudinarydotnet.1.1.1.nupkg
OK https://api.nuget.org/v3-flatcontainer/cloudinarydotnet/1.1.1/cloudinarydotnet.1.1.1.nupkg 509ms
Installing CloudinaryDotNet 1.1.1.
Adding package 'CloudinaryDotNet.1.1.1' to folder 'c:\users\user\source\repos\ConsoleApp1\packages'
Added package 'CloudinaryDotNet.1.1.1' to folder 'c:\users\user\source\repos\ConsoleApp1\packages'
Added package 'CloudinaryDotNet.1.1.1' to 'packages.config'
Successfully installed 'CloudinaryDotNet 1.1.1' to ConsoleApp1
Executing nuget actions took 2.27 sec
Time Elapsed: 00:00:03.3766194
========== Finished ==========
```

Could you please share a log that could help us understand the issue?

Thanks
Roee

**Comment by doug-shontz** [comment]
Let me know if there is something else I can get you.  As a bit of history...the library referencing Cloudinary's NuGet package was recently completely rebuilt in which we brought all of our internal class libraries up to .NET 4.7.1.  So, this library has never "been" another version.  We are using PackageReference as our NuGet update method.

Below is a stripped down version of our csproj file with anything "personal" taken out, and, below that is the error that NuGet throws when we attempt to update.

/*********************** CSPROJ *************************/

<?xml version="1.0" encoding="utf-8"?>
<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
  <PropertyGroup>
    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
    <ProjectGuid>{1478E178-97FA-40BD-A705-1B37125C9F69}</ProjectGuid>
    <OutputType>Library</OutputType>
    <AppDesignerFolder>Properties</AppDesignerFolder>
    <TargetFrameworkVersion>v4.7.1</TargetFrameworkVersion>
    <FileAlignment>512</FileAlignment>
  </PropertyGroup>
  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
    <DebugSymbols>true</DebugSymbols>
    <DebugType>full</DebugType>
    <Optimize>false</Optimize>
    <OutputPath>bin\Debug\</OutputPath>
    <DefineConstants>DEBUG;TRACE</DefineConstants>
    <ErrorReport>prompt</ErrorReport>
    <WarningLevel>4</WarningLevel>
  </PropertyGroup>
  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
    <DebugType>pdbonly</DebugType>
    <Optimize>true</Optimize>
    <OutputPath>bin\Release\</OutputPath>
    <DefineConstants>TRACE</DefineConstants>
    <ErrorReport>prompt</ErrorReport>
    <WarningLevel>4</WarningLevel>
  </PropertyGroup>
  <ItemGroup>
    <Reference Include="System" />
    <Reference Include="System.Configuration" />
    <Reference Include="System.Core" />
    <Reference Include="System.Drawing" />
    <Reference Include="System.Web" />
    <Reference Include="System.Web.Helpers, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
    <Reference Include="System.Web.Http, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
    <Reference Include="System.Xml.Linq" />
    <Reference Include="System.Data.DataSetExtensions" />
    <Reference Include="Microsoft.CSharp" />
    <Reference Include="System.Data" />
    <Reference Include="System.Xml" />
  </ItemGroup>
  <ItemGroup>
    <PackageReference Include="CloudinaryDotNet">
      <Version>1.1.0</Version>
    </PackageReference>
    <PackageReference Include="GoogleMapsApi">
      <Version>0.69.0</Version>
    </PackageReference>
    <PackageReference Include="ImageProcessor">
      <Version>2.6.0</Version>
    </PackageReference>
    <PackageReference Include="LaunchDarkly.Client">
      <Version>4.0.0</Version>
    </PackageReference>
    <PackageReference Include="Microsoft.Azure.NotificationHubs">
      <Version>1.0.9</Version>
    </PackageReference>
    <PackageReference Include="morelinq">
      <Version>2.10.0</Version>
    </PackageReference>
    <PackageReference Include="NEST">
      <Version>5.6.0</Version>
    </PackageReference>
    <PackageReference Include="NetEncryptionLibrary">
      <Version>4.0.0</Version>
    </PackageReference>
    <PackageReference Include="Newtonsoft.Json">
      <Version>10.0.3</Version>
    </PackageReference>
    <PackageReference Include="Sendgrid">
      <Version>9.9.0</Version>
    </PackageReference>
    <PackageReference Include="System.Net.Http">
      <Version>4.3.3</Version>
    </PackageReference>
    <PackageReference Include="ThinkMinistry.RESTAPIWrapper">
      <Version>3.1.19.32355</Version>
    </PackageReference>
    <PackageReference Include="Twilio">
      <Version>5.9.7</Version>
    </PackageReference>
    <PackageReference Include="Unity">
      <Version>5.6.1</Version>
    </PackageReference>
  </ItemGroup>
  <ItemGroup>
    <None Include="app.config" />
  </ItemGroup>
  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
</Project>

/********************* NUGET OUTPUT *************************/

Package CloudinaryDotNet 1.1.1 is not compatible with net471 (.NETFramework,Version=v4.7.1). Package CloudinaryDotNet 1.1.1 supports: netstandard1.6 (.NETStandard,Version=v1.6)
Package CloudinaryDotNet 1.1.1 is not compatible with net471 (.NETFramework,Version=v4.7.1) / win. Package CloudinaryDotNet 1.1.1 supports: netstandard1.6 (.NETStandard,Version=v1.6)
Package CloudinaryDotNet 1.1.1 is not compatible with net471 (.NETFramework,Version=v4.7.1) / win-x64. Package CloudinaryDotNet 1.1.1 supports: netstandard1.6 (.NETStandard,Version=v1.6)
Package CloudinaryDotNet 1.1.1 is not compatible with net471 (.NETFramework,Version=v4.7.1) / win-x86. Package CloudinaryDotNet 1.1.1 supports: netstandard1.6 (.NETStandard,Version=v1.6)

**Comment by doug-shontz** [comment]
Sorry...forgot the editor would pick up all the csproj stuff.  Attached this time.
[csproj.txt](https://github.com/cloudinary/CloudinaryDotNet/files/1804178/csproj.txt)

**Comment by roeeba** [comment]
Hi @doug-shontz. We've just released version 1.2.0 that fixes this issue. Please let me know if everything is ok on your side now?

**Comment by doug-shontz** [comment]
Was just able to upgrade the nuget package.  Still need to actually test our code, but we're not blocked by this issue any longer.  Thanks!
